src/xunit.console.uwp/xunit.console.uwp.csproj

```diff
@@ -42,12 +42,12 @@
     <Reference Include="Windows">
       <HintPath>C:\Program Files (x86)\Windows Kits\8.1\References\CommonConfiguration\Neutral\Windows.winmd</HintPath>
     </Reference>
-    <Reference Include="xunit.abstractions">
-      <HintPath>..\..\packages\xunit.abstractions.2.0.0\lib\net35\xunit.abstractions.dll</HintPath>
+    <Reference Include="xunit.abstractions, Version=2.0.0.0, Culture=neutral, PublicKeyToken=8d05b1bb7a6fdb6c, processorArchitecture=MSIL">
```

Comment:
@crummel can probably answer that.  I'd guess because when he started this using project.json in desktop 4.x apps was less common.